README.md

```diff
@@ -1,49 +1,65 @@
 SC Controller [![Build Status](https://travis-ci.org/kozec/sc-controller.svg?branch=master)](https://travis-ci.org/kozec/sc-controller)
 =============
 
-User-mode driver, mapper and GTK3 based GUI for Steam Controller, DS4 and similar controllers.
+User-mode driver, mapper and GTK3 based GUI for Steam Controller, DS4, Xbox 360 and similar controllers.
```

Comment:
Just a small issue here, Xbox 360 controller is not actually supported. Even adding it as evdev device would probably end bad, as sc-controller would start picking inputs from itself.

Reply:
I got my Xbox360 controller to work pretty well, but I see to get this updated in short